types/src/block_info.rs

```diff
@@ -155,10 +154,11 @@ impl Display for BlockInfo {
     fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
         write!(
             f,
-            "BlockInfo: [epoch: {}, round: {}, id: {}, version: {}, timestamp (us): {}, next_validator_set: {}]",
+            "BlockInfo: [epoch: {}, round: {}, id: {}, executed_state_id: {}, version: {}, timestamp (us): {}, next_validator_set: {}]",
```

Comment:
why aren't we calling this root_hash?

Reply:
That's how consensus call it. cc: @zekun000